app/post/modify/post-editor.directive.js

```diff
@@ -49,8 +50,10 @@ function PostEditorController(
     UserEndpoint,
     Notify,
     _,
-    PostActionsService
+    PostActionsService,
+    $rootScope
   ) {
+    $rootScope.setLayout('layout-a');
```

Comment:
Can you use the `layout-class` directive instead? I'm not keen on requiring $rootScope everywhere.